src/main/resources/fi/metatavu/metamind/changelog.xml

```diff
@@ -387,12 +387,29 @@
     </addColumn>
   </changeSet>
   
-  <changeSet id="default-hint-for-story" author="Daniil Smirnov">
+  <changeSet id="default-hint-for-story-fix" author="Daniil Smirnov">
```

Comment:
Entries that are already part of merged PR's in changelog.xml cannot be modified. This would cause a history rewrite that would break migrations.